projects/plugins/jetpack/class-jetpack-xmlrpc-methods.php

```diff
@@ -196,6 +197,23 @@ public static function json_api( $args = array() ) {
 		);
 	}
 
+	/**
+	 * Filters the response of the remote_provision XMLRPC method
+	 *
+	 * @param array $response The response.
+	 * @param array $request An array containing at minimum a nonce key and a local_username key.
+	 *
+	 * @since 9.8.0
+	 * @return array
+	 */
+	public static function remote_provision_response( $response, $request ) {
+		if ( isset( $request['onboarding'] ) && ! empty( $request['onboarding'] ) ) {
```

Comment:
I think you should be able to simplify this, since `! empty()` already handles `isset()`:

```suggestion
		if ( ! empty( $request['onboarding'] ) ) {
```